tests/classify_test.py

```diff
@@ -128,3 +128,23 @@ def test_application_directories(in_tmpdir, no_empty_path):
     # Should be application with extra directories
     ret = classify_import('testing', application_directories=('.', 'tests'))
     assert ret is ImportType.APPLICATION
+
+
+def test_override_absent_application_modules():
+    # Should be classified 3rd party without argument
+    ret = classify_import('c_module')
+    assert ret is ImportType.THIRD_PARTY
+    # Should be classified application with the override
+    ret = classify_import(
+        'c_module', override_absent_application_modules=('c_module'),
```

Comment:
just fyi -- this isn't a tuple it's a parenthesized string -- a 1-element tuple must have a trailing comma

no need to fix it, I'll handle it when I change the parameter name 👍

Reply:
So tests worked because `'fubar' in 'fubar'` is `True` like `'fubar' in ('fubar',)` and I'm guessing the typing didn't complain as a string is a container of strings?

Reply:
yeah the abc for container is ~essentially:

```python
class Container(Generic[T]):
    def __contains__(self, other: T) -> bool: ...
```

of which `str` satisfies `Container[str]`